frontend/src/metabase/visualizations/lib/renderer_utils.js

```diff
@@ -8,7 +8,10 @@ import { datasetContainsNoResults } from "metabase/lib/dataset";
 import { parseTimestamp } from "metabase/lib/time";
 
 import { dimensionIsNumeric } from "./numeric";
-import { dimensionIsTimeseries } from "./timeseries";
+import {
+  dimensionIsTimeseries,
```

Comment:
In light of this issue, `dimensionIsTimeseries` seems a little misleading, but I can't come up with a better name than `dimensionMightBeTimeseries` or `dimensionIsTimeseriesMaybe` or something.

Reply:
🤔 yeah good point. Maybe `dimensionLooksLikeTimeseries` if we changed to logic to just inspect the data, but then we'd need to do `dimensionIsTimeseries() || dimensionLooksLikeTimeseries()` in a few places to avoid actually naming that concept.